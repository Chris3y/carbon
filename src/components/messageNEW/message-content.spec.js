import React from 'react';
import { shallow } from 'enzyme';
import MessageContent from './message-content.component';
import 'jest-styled-components';

function render(props) {
  return shallow(<MessageContent { ...props } />);
}

describe('MessageContent', () => {
  describe('when rendered', () => {
    it('should match snapshot', () => {
      expect(render()).toMatchSnapshot();
    });
  });
});
