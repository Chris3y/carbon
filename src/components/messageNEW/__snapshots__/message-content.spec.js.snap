// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageContent when rendered should match snapshot 1`] = `
<styled.div
  transparent={false}
  type="info"
>
  <Content
    align="left"
    as="primary"
    bodyFullWidth={false}
    className="message-content"
    inline={false}
  />
</styled.div>
`;
